src/chainparams.cpp

```diff
@@ -125,12 +125,12 @@ class CMainParams : public CChainParams {
         assert(genesis.hashMerkleRoot == uint256S("0x4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b"));
 
         // Note that of those with the service bits flag, most only support a subset of possible options
```

Comment:
comment needs updating: there is no 'service bits flag' anymore

Reply:
Done, in a new commit.